vendor/modules.txt

```diff
@@ -565,6 +565,8 @@ google.golang.org/grpc/internal/transport/networktype
 google.golang.org/grpc/keepalive
 google.golang.org/grpc/metadata
 google.golang.org/grpc/peer
+google.golang.org/grpc/reflection
```

Comment:
There's no change in `go.mod`, is this correct?

Reply:
already included in google.golang.org/grpc